### ISSUE TITLE
Write a tutorial for computing kinematic variables

### ISSUE THREAD

**Issue by niksirbi** [neuroinformatics-unit/movement#114]
**Is your feature request related to a problem? Please describe.**
After #3 is merged, we need a way to document the new features.

**Describe the solution you'd like**
We can make a new "example", similar to the one we already have, made with `sphinx-gallery`.
The example will load one of our datasets,  and will walk users through computing the various kinematic variables, and going back-and-forth between cartesian and polar coordinate systems.

This should be also accompanied by adding the new functions to the API reference.

**Additional context**
We also need a "glossary" (see #69), where we give exact definitions of the variables we deal with and establish the nomenclature.  But this should/may be a separate PR.

**Comment by lochhh** [comment]
Closing as completed in #135
